buildNumber.properties

```diff
@@ -1,3 +1,3 @@
 #maven.buildNumber.plugin properties file
```

Comment:
This file should be excluded from commits.

We want to remove it in the future from the repository.